action.yml

```diff
@@ -18,7 +18,8 @@ inputs:
     default: fatal
   VALIDATE_LINKS:
     description: Validate hyperlinks
-    default: false
+    default: "false"
+    required: false
```

Comment:
These changes were because VSCode complains about the schema validation based on https://json.schemastore.org/github-action.json

Reply:
We should update this for all other inputs as a separate PR.